observation_portal/observations/test/tests.py

```diff
@@ -1640,6 +1648,148 @@ def test_with_one_obs_command_reports_dry_run_doesnt_modify_time(self):
         self.assertEqual(self.time_allocation.std_time_used, 0)
 
 
+class TestGetObservationsDetailAPIView(APITestCase):
```

Comment:
These tests, lines 1651 through 1792, are tests that I moved over from `observation_portal/observations/test_views.py` and refactored to test the observations API endpoints. Before they had been testing the `/observations` and `/observations/<obsId>` pages.